get_unit_test_data.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash --posix
+
+# grab select aws files stored in the bdp
```

Comment:
Should we store this in some dedicated place for unit test data that is specific to this repo? 
If we do, we might want to remove "omf_*" fields from these files to save on size? We might also want to somehow flag that this data is for specific unti test. Also maybe use aws syn instead of cp?

Reply:
Yeah, those are all good ideas.  My suggestion would be our bdp bucket for now as we won't necessarily want to dole out our aws credentials all the time.  We could create super small netcdf files containing only enough records to test the machinery.

Reply:
Ultimately, I believe this effort is valid and should be done, but it is outside the scope of this particular pull request.